toqito/measurements/tests/test_pretty_bad_measurement.py

```diff
@@ -0,0 +1,75 @@
+"""Test pretty_bad_measurement."""
+
+import numpy as np
+import pytest
+
+from toqito.measurements import pretty_bad_measurement
+from toqito.states import bell, trine
+
```

Comment:
I think you could add a test in the test files for both PGM and PBM to verify the output of the functions satisfies all POVM properties:

- all sum up to the identity or close to it (span the entire space)
- are orthogonal to each other, positive semi-definite

Reply:
Hmm, I'm not sure about adding these extra tests. In some way, this would be testing the functionality of the convex optimization solvers. Since the constraints themselves enforce this property, it may be redundant to include tests on top of that to check if the measurements (the variables that are returned from the SDP) are valid POVMs. But then again, I can see an argument on the other side as well as we check the optimal value. I'm probably lean towards not adding this, but let me know what you think.